src/ui/mixins/context-menu-gesture.js

```diff
@@ -35,7 +40,13 @@ export default {
             let actions = actionsCollection.getVisibleActions();
             let sortedActions = this.openmct.actions._groupAndSortActions(actions);
 
-            this.openmct.menus.showMenu(event.clientX, event.clientY, sortedActions);
+            this.openmct.menus.showMenu(event.clientX, event.clientY, sortedActions, this.onContextMenuDestroyed);
+            this.contextClickEnabled = true;
+            this.$emit('context-click-enabled', true);
```

Comment:
Why do you have to emit this? doesn't the contextClickEnabled property exist on whoever us using the mixin?

Reply:
It’s needed in tree item since the mixin is used in object label

Reply:
Ahh, ok gotcha, thanks!